tests/test_modeling_flax_common.py

```diff
@@ -168,6 +169,7 @@ def recursive_check(tuple_object, dict_object):
             dict_inputs = self._prepare_for_class(inputs_dict, model_class)
             check_equivalence(model, tuple_inputs, dict_inputs, {"output_hidden_states": True})
 
+    # (Copied from tests.test_modeling_common.ModelTesterMixin.check_outputs)
```

Comment:
The intention is only to add this information, but mean to work with the current version of `make fix-copies`.

@sgugger Are you OK with this comment? Otherwise I can just remove it.